packages/zoe/src/zoeService/invitationQueries.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+
+export const makeInvitationQueryFns = invitationIssuer => {
+  /** @type {GetInvitationDetails} */
+  const getInvitationDetails = async invitationP => {
+    const onRejected = reason => {
+      const err = assert.error(X`A Zoe invitation is required, not ${invitationP}`);
```

Comment:
Sorry, I should have run `yarn lint-fix` before merging my suggestion into your PR.

Reply:
No worries, thanks for your PR!